### DISCUSSION TITLE
Sandbox environment for testing and development

### DISCUSSION THREAD

**Discussion by tzehetner** [keygen-sh/community#119] [Q&A] [answered]
Hi,

I have some questions about environments:

- Can environments only be created via api?
- If I create an isolated sandbox environment, does this environment have its own dashboard, so that test data doesn't pollute my production dashboard? Or should I signup for another account, that I use for testing during development?

Thanks!

**Comment by ezekg** [comment, marked as answer]
Right now, the UI doesn't support environments. Environments are an API-only feature at the moment. But we're working on a brand new Portal app that _will_ support environments, and each environment will have its own metrics dashboard functionality. There are no current plans to add environments to the current dashboard app, since it will be retired when our new Portal app launches.

**Reply by ezekg** [reply]
If you need a separate dev account for now, feel free to open one.

**Reply by tzehetner** [reply]
Thanks for the quick response. I will use a second account until the new portal is available.
